app/models/reservation_scopes.rb

```diff
@@ -56,32 +42,29 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :ends_on_days, lambda { |start_date, end_date|
         where(due_date: start_date..end_date)
       }
-      scope :reserved_on_date, ->(date) { overlaps_with_date(date).finalized }
+      scope :reserved_on_date, lambda { |date|
+        overlaps_with_date(date).not_available
+      }
       scope :for_eq_model, lambda { |eq_model|
-        where(equipment_model_id: eq_model.id).finalized
+        where(equipment_model_id: eq_model.id)
       }
       scope :active_or_requested, lambda {
-        where('checked_in IS NULL and approval_status != ?', 'denied').recent
+        where(status: Reservation.statuses.values_at(
```

Comment:
Similar to `not_available`, I feel like this could be modified to `where(status: [:requested, :reserved, :checked_out])`.